### PR TITLE
update to use momentum for cookbook rollout

### PR DESCRIPTION
**Pull request by izakp**

### CODE REVIEW

File: ci/prepare.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  tar xzf otp_src_$ERLANG_VERSION.tar.gz
+  cd $ERLANG_PATH
+  ./configure --enable-smp-support \
+              --enable-m64-build \
+              --disable-native-libs \
+              --disable-sctp \
+              --enable-threads \
+              --enable-kernel-poll \
+              --disable-hipe \
+              --without-javac
```
Comment (joeyAghion) [inline ci/prepare.sh:35]: Mental note to myself: look these up

File: ci/tests.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export MIX_ENV="test"
```
Comment (joeyAghion) [inline ci/tests.sh]: Sort of surprised this is necessary, since I thought it was set by the `mix test` command already.

Comment (izakp) [inline ci/tests.sh]: Ahh i see what you mean

File: ci/prepare.sh [hunk 3]
```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Lifted from https://gist.github.com/joakimk/48ed80f1a7adb5f5ea27
+# until CircleCI adds official Elixr/Erlang support https://discuss.circleci.com/t/add-support-for-elixir-erlang/2215
+
+set -e
+
+export ERLANG_VERSION="17.5"
```
Comment (izakp) [inline ci/prepare.sh]: @ashkan18 are these versions OK?

File: ci/prepare.sh (hunk 1 above)
Comment (ashkan18) [inline ci/prepare.sh:35]: Mental note to myself: always look at Joey's mental notes and follow those.

**Comment by ashkan18** [comment]
Thanks @izakp

**Comment by joeyAghion** [comment]
Nice work!

**Pull request merged**